angular/projects/spark-angular/src/lib/components/sprk-masthead/sprk-masthead-selector/sprk-masthead-selector.component.ts

```diff
@@ -75,9 +77,7 @@ import { ISprkMastheadSelectorChoice } from '../sprk-masthead-selector/sprk-mast
         <ul
           class="sprk-c-Dropdown__links"
           role="listbox"
-          [attr.aria-label]="
-            heading ? heading : screenReaderText || 'My Choices'
-          "
+          [attr.aria-label]="'My Choices'"
```

Comment:
Note: Talked about this in chat -- in production, because there hasn't been a way in masthead to set screenReaderText or heading, it always defaulted to "My Choices".

To wrangle in scope, we're going to address this in another issue. @ssabo1  knows about this and will create an issue.